src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_cython.pyx

```diff
@@ -560,7 +557,12 @@ cdef class PyDBFrame:
 
         return stopped
 
-    def get_func_name(self, frame):
+    # IFDEF CYTHON -- DONT EDIT THIS FILE (it is automatically generated)
```

Comment:
Note: while reviewing the code in this module I ended up doing a bunch of minor cython improvements.